src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.html

```diff
@@ -0,0 +1,57 @@
+<cd-modal [modalRef]="activeModal">
```

Comment:
A better approach would be to implement the form as a separate component and show it as a model dialog via the [ModalService](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/shared/services/modal.service.ts) or by using the [FormModalComponent](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/shared/components/form-modal/form-modal.component.ts#L18) to define the form in a declarative way.